src/flameGraph.h

```diff
@@ -71,7 +71,7 @@ class Trie {
 class FlameGraph {
   private:
     Trie _root;
-    std::map<std::string, u32> _cpool;
+    std::unordered_map<std::string, u32> _cpool;
```

Comment:
Not that simple :) This task requires thoughtful decision what occurrences of `map` should be changed and what should not.

Reply:
Keeping strings in a constant pool sorted alphabetically pursues two goals:
1. Consistent order of frames on a flamegraph.
2. Smaller html size due to [string compression](https://github.com/async-profiler/async-profiler/blob/ee75d80622a54e8919052cdd195fd49c8977c755/src/res/flame.html#L204-L208).

Reply:
👍 I plan to profile the profiler (I got some resources from @krk) to check whether any paths became less/more efficient. That should give some guidance on the access patterns for each occurrent of `std::map`.

Reply:
Sounds good! Also check how changes affect static size of the binary.